src/TileManifest.css

```diff
@@ -1,26 +1,23 @@
 .TileManifest {
-  display: grid;
-  grid-auto-rows: 0.5in;
-  grid-auto-flow: row dense;
-  gap: 1px;
-  grid-template: "title title title title" 0.5in / repeat(
-      4,
-      calc((8in - 5px) / 4)
-    );
```

Comment:
This setting of grid-template doesn't work for two reasons:

1. wrong syntax
2. width calculations are made on Tile level and overwriting grid-template setting